### ISSUE TITLE
 v118.1 broken compressed textures

### ISSUE THREAD

**Issue by Usnul** [mrdoob/three.js#19742]
##### Description of the problem

Migrated to v118.1, gltf model with compressed textures stopped rendering it's material, here's the error:
![image](https://user-images.githubusercontent.com/5939063/85894113-6b666f80-b7f4-11ea-9dff-206868d56585.png)

I suspect it's related to https://github.com/mrdoob/three.js/pull/16287

here's what that looks like:
![image](https://user-images.githubusercontent.com/5939063/85894177-8cc75b80-b7f4-11ea-8215-92cad5ba1293.png)

just to be clear, the black things are the models with compressed textures. Here's a snippet of what those look like in gltf (only relevant parts):
```json
{
        "extensionsUsed":[
		"MSFT_texture_dds"
	],
	"images" : [
        {
            "name" : "diffuse", 
            "uri" : "props1_512.png"
        },
        {
            "name" : "normal", 
            "uri" : "props1_n_512.png"
        },
        {
            "name" : "metalness", 
            "uri" : "props1_arm_512.png"
        },
		{
			"uri": "props1_512_dtx5.dds"
		},
		{
			"uri": "props1_n_512_dtx5.dds"
		},
		{
			"uri": "props1_arm_512_dtx5.dds"
		}
    ],
	"textures" : [
        {
            "source" : 0,
			"extensions": {
            "MSFT_texture_dds": {
					"source": 3
				}
			}
        },
        {
            "source" : 1,
			"extensions": {
				"MSFT_texture_dds": {
					"source": 4
				}
			}
        },
        {
            "source" : 2,
			"extensions": {
				"MSFT_texture_dds": {
					"source": 5
				}
			}
        }
    ],
    "materials" : [
        {
            "name" : "Props1",
            "normalTexture" : {
                "index" : 1,
                "scale" : 1,
                "texCoord" : 0
            },
            "pbrMetallicRoughness" : {
                "baseColorFactor" : [
                    1, 
                    1, 
                    1, 
                    1.0
                ], 
                "baseColorTexture" : {
                    "index" : 0
                },
                "metallicRoughnessTexture" : {
                    "index" : 2
                }
            }
        }
    ]
}
```

##### Three.js version

- [ ] Dev
- [x] r118
- [ ] ...

##### Browser

- [ ] All of them
- [x] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [ ] All of them
- [x] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

**Comment by Mugen87** [comment]
The VAO change is unrelated to this issue. 

It's fixed via #19738.

**Comment by Mugen87** [comment]
Duplicate of #19734.
